File: java/FacebookSlim/facebookslim/src/main/java/com/tonestro/facebookslim/AppEventsLoggerSlim.java
package com.tonestro.facebookslim;

import android.content.Context;
import android.os.Bundle;

import com.facebook.appevents.AppEventsLogger;

public class AppEventsLoggerSlim {
    private final AppEventsLogger logger;

    private AppEventsLoggerSlim(Context context) {
        logger = AppEventsLogger.newLogger(context);
    }

    public static AppEventsLoggerSlim newLogger(Context context) {
        return new AppEventsLoggerSlim(context);
    }

    public void logEvent(String eventName, Bundle parameters) {
        logger.logEvent(eventName, parameters);
    }
}

File: Xamarin.FacebookSlim.Droid.Dummy/AppEventsLoggerSlim.cs
using System;
namespace Xamarin.FacebookSlim.Droid.Dummy
{
    public class AppEventsLoggerSlim
    {
        public static AppEventsLoggerSlim NewLogger(object context)
        {
            return new AppEventsLoggerSlim();
        }

        public void LogEvent(string appEvent, object ventProperties) {
           
        }
    }
}

